services/brig/test/integration/API/TURN.hs

```diff
@@ -165,14 +163,29 @@ toTurnURI s h p t = turnURI s ip p t
     ip = fromMaybe (error "Failed to parse host address")
        $ fromByteString h
 
-setTurn :: FilePath -> String -> IO ()
-setTurn cfgDest newConf = do
-    tmpDir <- getTemporaryDirectory
-    tmpPathFile <- writeTempFile tmpDir "-turn.tmp" newConf
-    copyFile tmpPathFile cfgDest
+
+type TurnUpdater = String -> IO ()
+
+withTurnFile :: FilePath -> (TurnUpdater -> Http ()) -> Http ()
+withTurnFile cfgDest action = do
+    tmpdirRoot <- lookupTempDir
+    Temp.withTempDirectory tmpdirRoot "wire.temp" $ \tempdir -> do
```

Comment:
I am unable to compile things right now and check this :) 
Question: does it make sense to `tmpdirRoot <- lookupTempDir` -> wouldn't it be easier to simply use `withSystemTempDirectory`?

Reply:
yes, i didn't think of that.  will change, thanks!